pkg/maps/ctmap/ctmap.go

```diff
@@ -287,6 +301,10 @@ func doGC4(m *bpf.Map, filter *GCFilter) int {
 		return 0
 	}
 
+	if filter.fType&GCFilterByID != 0 && len(filter.IDsToRm) == 0 {
```

Comment:
I've seen this a couple of times, perhaps we could express this with a function like the below, for improved readability:
```
func (filter *GCFilter) hasNoIDsToRemove() bool
```

Reply:
Did you see it more than 2 times? I've added the function but want to make sure I didn't miss any spot.

Reply:
I just saw it twice.